recipe/bld.bat

```diff
@@ -0,0 +1,12 @@
+mkdir build	
```

Comment:
We may also need this...

```suggestion
set "PROTOBUF_USE_DLLS="
mkdir build	
```

https://github.com/protocolbuffers/protobuf/blob/214c77e1b76e63e512bd675d1c300c80438642b6/cmake/README.md#dlls-vs-static-linking
https://github.com/protocolbuffers/protobuf/issues/2502#issuecomment-412868718